bench-vortex/src/medicare_data.rs

```diff
@@ -0,0 +1,87 @@
+use std::fs::File;
+use std::io::BufReader;
+use std::path::PathBuf;
+use std::sync::Arc;
+
+use arrow::datatypes::SchemaRef;
+use arrow_csv::reader::Format;
+use itertools::Itertools;
+use vortex::array::chunked::ChunkedArray;
+use vortex::array::IntoArray;
+use vortex::arrow::FromArrowType;
+use vortex::serde::WriteCtx;
+use vortex_error::VortexError;
+use vortex_schema::DType;
+
+use crate::data_downloads::{decompress_bz2, download_data, parquet_to_lance};
+use crate::idempotent;
+use crate::reader::{compress_csv_to_vortex, default_csv_format, write_csv_as_parquet};
+
+pub fn medicare_data_csv() -> PathBuf {
+    let fname = "Medicare1_1.csv.bz2";
+    download_data(
+        fname,
+        "http://www.cwi.nl/~boncz/PublicBIbenchmark/Medicare1/Medicare1_1.csv.bz2",
```

Comment:
FLUP, but we should probably centralize our testing data in a place we own (maybe just a public S3/R2 bucket)

Reply:
Yeah I agree that we should just grab a copy of all of the public BI benchmark data and host it ourselves. That way if their website is down and/or goes offline and/or changes we have a canonical copy.